### ISSUE TITLE
mjunoon plugin  streamlink stop after some time

### ISSUE THREAD

**Issue by maanich** [streamlink/streamlink#1911]
Hi all  i added live streams from mjuoon website and make them as script to run together but streamlink shutdown after some time must restart streamlink to see streams in vlc.

script i made like this 
#!/bin/bash
#############################
killall streamlink
streamlink --player-continuous-http --player-external-http --player-external-http-port=9200 https://www.mjunoon.tv/such-tv-live best > stdoutfile 2> stderrfile &
streamlink --player-continuous-http --player-external-http --player-external-http-port=9202 https://www.mjunoon.tv/express-news-live best > stdoutfile 2> stderrfile &
streamlink --player-continuous-http --player-external-http --player-external-http-port=9201 https://www.mjunoon.tv/a-plus-live best > stdoutfile 2> stderrfile &

n run http://ip:port  in vlc
 can we make autostart script if streamlink goes down it start automatically.
Regards.

**Comment by beardypig** [comment]
Any chance of following the issue template this time? 

Please provide the debug logs. 
How much time is “some time”? 10 seconds, 5 minutes, 10 minutes, 6 hours?
Perhaps try on a desktop with a player like VLC or mpv and see it still stops.

**Comment by maanich** [comment]
i added all links in a script and run but it goes some time 1 hrs some time 30 min some time less can i add a log option in script with in the line so we know when it crash.

**Comment by gravyboat** [comment]
@maanich If you can't follow the template and provide the required information we cannot, and will not help you. You've been warned several times about this and are creating a ton of issues. We also don't provide support for custom scripts of any kind. You will need to provide debug output from Streamlink commands only.

**Comment by gravyboat** [comment]
When you create a new issue you can select the template you want to use. There are 4 available and you should select the appropriate one then fill everything out that it asks for.

**Comment by maanich** [comment]
i select no.3 plugin issue and typed the details.

**Comment by beardypig** [comment]
@maanich did you delete everything that was automatically filled it?

**Comment by maanich** [comment]
yes i did    just added three example streamlinks the way i try to use. simple stuff  i didnot find any thihg wrong if u got any thing u can add screen shot of it so i know where i am wrong and how i addee that filled stuff.

**Comment by beardypig** [comment]
You need to read the text in the box when you create an issue. It explains all the information we _need_ you to give us to help you. For example the debug logs (or any logs), which you still haven’t provided.
